pandas/tests/series/test_analytics.py

```diff
@@ -1242,16 +1242,32 @@ def test_idxmin(self):
         result = s.idxmin()
         assert result == 1
 
-    def test_numpy_argmin(self):
-        # argmin is aliased to idxmin
-        data = np.random.randint(0, 11, size=10)
-        result = np.argmin(Series(data))
-        assert result == np.argmin(data)
+    def test_numpy_argmin_deprecated(self):
+        # See gh-16830
+        data = np.arange(10)
+
+        s = Series(data)
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+            # The deprecation of Series.argmin also causes a deprecation
+            # warning when calling np.argmin. This behavior is temporary
+            # until the implemention of Series.argmin is corrected.
+            result = np.argmin(s)
+
+        assert result == 0
```

Comment:
why are you asserting here directly to 0, and below to the result of `expected = s.idxmin()`. Both can use the same expected value? (I see in the recent commits you changed this)

Reply:
I just changed them all to compare directly to scalars. I think that's preferable.